### PR TITLE
Update examples to use updated space drawing

### LINKED ISSUE #2432: Update examples to use cleaned up matplotlib space plotting
With #2430 merged, all examples should be checked and updated to correctly use the new code. Wolf sheep is already up to date and can serve as a template.

### PR DESCRIPTION
**Pull request by quaquel**
This updates the examples to use the new space drawing functionality properly.

I actually think that sugarscape can now be fully expressed without using a custom space drawing function, but I have left that for now since its convenient to have an example that shows how custom space drawing is possible. 

Closes #2432

### CODE REVIEW

File: mesa/examples/advanced/epstein_civil_violence/app.py [hunk 1]
```diff
@@ -36,6 +36,13 @@ def citizen_cop_portrayal(agent):
     return portrayal
 
 
+def post_process(ax):
+    ax.set_aspect("equal")
+    ax.set_xticks([])
+    ax.set_yticks([])
+    ax.get_figure().set_size_inches(10, 10)
```
Comment (EwoutH) [inline mesa/examples/advanced/epstein_civil_violence/app.py:43]: I really like this possibility!

**Review by EwoutH — APPROVED** [review]
Good stuff. Could you add screenshots how they now look?

**Comment by quaquel** [comment]
The only one that really is changed is virus on a network. The others are mainly cleaning up of code and minor visual tweaks (e.g. set_aspect('equal') on axes.


Screenshot 2024-10-30 at 20 07 07

File: mesa/examples/advanced/epstein_civil_violence/app.py (hunk 1 above)
Comment (quaquel) [reply mesa/examples/advanced/epstein_civil_violence/app.py:43]: yes it gives a lot more control over visual styling

**Pull request merged**